source/backend/dal/Constants/FormTypeCode.cs

```diff
@@ -1,6 +1,6 @@
 namespace Pims.Dal.Constants
 {
-    public static class DocumentFormType
+    public static class FormTypeCode
```

Comment:
Is it our normal pattern to have 2 enums for a single type on the backend? one for the dal and one for the api?

Reply:
Well, one is not an enum. but we do have this sometimes